### PR TITLE
[MRG] fix for add_poisson_drive not working with int when cell_specific=True

### PR DESCRIPTION
**Pull request by dylansdaniels**
This fix addresses issue #816 and updates the test for adding poisson drives in test_drives.py with additional conditions to check

### CODE REVIEW

File: hnn_core/tests/test_drives.py [hunk 1]
```diff
@@ -469,6 +469,7 @@ def test_drive_random_state():
 
 @pytest.mark.parametrize("rate_constant,cell_specific,n_drive_cells",
                          [(2, False, 1), (2.0, False, 1),
+                          (2, False, 1), (2.0, False, 1),
```
Comment (jasmainak) [inline hnn_core/tests/test_drives.py:472]: needs to be updated

Comment (dylansdaniels) [reply hnn_core/tests/test_drives.py]: Woops 😆 fixed now!

**Comment by jasmainak** [comment]
Thanks @dylansdaniels !!!

**Comment by rythorpe** [comment]
FYI @dylansdaniels, if you use "closes" or "fixes" followed by the issue tag in the first comment of your PR, GH will close the issue automatically once the PR is merged. :partying_face:

**Pull request merged**